src/styles/card.scss

```diff
@@ -0,0 +1,90 @@
+.card {
+  width: $card-width;
+  padding: 32px 16px 16px;
+  background-color: #fff;
```

Comment:
Lines 4,5,79,86 may be better with variables

Reply:
If you use this value several times - move it to the variable